app/furniture/marketplace/delivery_areas/index.html.erb

```diff
@@ -19,5 +23,12 @@
           scheme: :secondary) %>
       <%- end %>
     </div>
+    <div class="text-center mt-3">
+      <%- if params[:archive] %>
+        <%= link_to "Active", marketplace.location(child: :delivery_areas) %>
+      <%- else %>
+        <%= link_to "Archive", marketplace.location(child: :delivery_areas, query_params: { archive: true }) %>
```

Comment:
Maybe the links want to be say "Active areas" and "Archived areas" (or "Active delivery areas", if there is enough space), otherwise I'm not sure it's clear what they are for.

Reply:
Nice, I've updated these too!